### DISCUSSION TITLE
Ordered list stops numbering at 20

### DISCUSSION THREAD

**Discussion by Hoverth** [nvim-neorg/neorg#1409] [Q&A] [answered]
Using the ordered list (`~`) stops counting when it hits 20, and just defaults back to the tilde. Is there any way to use just normal number characters instead of the fancy ones? There seems to be something in the concealer wiki page on it, but I can't figure out how to get it working.
Any help would be greatly appreciated, sincerely, someone who needs long lists

**Comment by Hoverth** [comment]
I have managed to make it use actual numbered characters, but it only goes up to 9 now
```lua
          ["core.concealer"] = {
            config = {
              icons = {
                ordered = {
                  icons = {"1", "a", "i", "A", "I"},
                },
              },
            },
          },
```
Is there any way I can make it actually count up properly?

**Reply by katawful** [reply]
neovim doesn't support concealing 1 character into 2 characters. you're stuck with font limitations until that is possible

**Reply by Hoverth** [reply]
ah. that sucks.

thanks anyway!

**Reply by max397574** [reply]
neovim actually does support that know with anticonceal

**Reply by Hoverth** [reply]
oh! does it just need to be implemented or something?

**Reply by katawful** [reply]
https://github.com/neovim/neovim/pull/9496

unless i'm missing something, anticonceal still isn't supported

**Reply by max397574** [reply]
@katawful https://github.com/neovim/neovim/pull/20130

**Reply by katawful** [reply]
that's not concealing though? you can't interact with virtual text, which is rather important for lists

**Reply by max397574** [reply]
![image](https://github.com/nvim-neorg/neorg/assets/81827001/2f888250-a4e4-4b1a-a92e-f32e83e25161)
this it out of the box for me
you can just move your cursor to the first character of the number which is perfectly fine imo since in neorg you don't modify the numbers yourself anyway

**Reply by Hoverth** [reply]
@max397574 can you share your configuration please? I would really like to get something similar to how it works for you.

**Reply by max397574** [reply, marked as answer]
works out of the box on nightly neovim

**Reply by Hoverth** [reply]
Hmm I'm on arch's v0.9.5, maybe it's a yet to be released feature or something.

Thanks anyway!
